endpoint_info.go

```diff
@@ -149,6 +151,33 @@ func (ep *endpoint) InterfaceNames() []driverapi.InterfaceNameInfo {
 	return iList
 }
 
+func (ep *endpoint) SetStaticRoutes(routes []types.StaticRoute) error {
+	ep.Lock()
+	defer ep.Unlock()
+
+	for _, r := range routes {
+		if r.RouteType == types.NEXTHOP {
+			ep.joinInfo.StaticRoutes = append(ep.joinInfo.StaticRoutes, &r)
+		} else {
+			if err := ep.AddInterfaceRoute(&r); err != nil {
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+func (ep *endpoint) AddInterfaceRoute(route *types.StaticRoute) error {
```

Comment:
since this is not going to an exported api, can you make `AddInterfaceRoute` not exported?